### PR TITLE
chore: update error codes for binanceusdm.ts [ccxt-pro]

### PR DESCRIPTION
**Pull request by ChristianCoenen**
In https://github.com/ccxt/ccxt/pull/17640, I updated the error codes for binance-usdm. This change must also be added to the `pro/binanceusdm.ts` file.

Are there any plans to remove this redundancy?

### CODE REVIEW

File: ts/src/pro/binanceusdm.ts [hunk 1]
```diff
@@ -17,6 +18,14 @@ export default class binanceusdm extends binance {
                 'fetchMarkets': [ 'linear' ],
                 'defaultSubType': 'linear',
             },
+            // https://binance-docs.github.io/apidocs/futures/en/#error-codes
+            'exceptions': {
```
[the remark this answers is not in the document]
Comment (ChristianCoenen) [reply ts/src/pro/binanceusdm.ts:22]: This is not the one that is added. The one added is a -5021 Invalid Order for USDM. The error code -5021 is the one from `binance.ts` (spot) which is a different error. (yes -5021 exists twice depending whether you're on spot or usdm)

[the remark this answers is not in the document]
Comment (ChristianCoenen) [reply ts/src/pro/binanceusdm.ts:22]: You can verify this by looking for -5022 or -5028. I doesn't exist.

[the remark this answers is not in the document]
Comment (carlosmiei) [reply ts/src/pro/binanceusdm.ts:22]: @ChristianCoenen my bad than, let me re-check

**Comment by carlosmiei** [comment]
LGTM ✅

**Review by carlosmiei — APPROVED** [review]

**Pull request merged**